### PR TITLE
Support changing work owners.

### PR DESCRIPTION
**Pull request by justinlittman**
refs #2312

## Why was this change made? 🤔
Support admin function.

## How was this change tested? 🤨

⚡ ⚠ If this change involves consuming from or writing to another service (or shared file system), ***run [integration test create_object_h2_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test manually in [stage|qa] environment, in addition to specs. ⚡

Unit

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -15,7 +15,10 @@ class ApplicationController < ActionController::Base
   authorize :user_with_groups
 
   def user_with_groups
-    UserWithGroups.new(user: current_user, groups: ldap_groups) if current_user
+    UserWithGroups.new(user: current_user, groups: ldap_groups)
+  rescue Devise::MissingWarden
+    # A nil UserWithGroups, indicating that not logged in
```
Comment (jcoyne) [inline app/controllers/application_controller.rb:20]: This should never happen.   If we are seeing this. Then components will not be showing up on the page (after a live update) when they should

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (jcoyne) [reply app/controllers/application_controller.rb:20]: Can this change be reverted?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (justinlittman) [reply app/controllers/application_controller.rb:20]: What? I thought I did. Looking.

Comment (justinlittman) [reply app/controllers/application_controller.rb]: Apparently it only works if you hit save in VSCode ...

**Review by jcoyne — APPROVED** [review]

**Pull request merged**